ethereum/p2p/build.gradle

```diff
@@ -71,4 +72,5 @@ dependencies {
   testImplementation 'org.assertj:assertj-core'
   testImplementation 'org.awaitility:awaitility'
   testImplementation 'org.mockito:mockito-core'
+  testImplementation 'org.springframework:spring-core'
```

Comment:
What is spring used for?

Reply:
It is used in the TLSContextFactoryTest testcase to SocketUtils.findAvailableTcpPort

Reply:
any chance to use something else? Usually it's just a socket with port 0 that is opened, closed, and the port is recycled. Better even if the server supports port 0.

Reply:
done, i have removed the spring dep and am using the server bind with 0 and then get the allocated port